src/lib/amount.ts

```diff
@@ -334,20 +334,26 @@ export function setRequiredAmount(
 
     const fee = new BN(1024).mul(precision_multiplier).div(new BN(1021));
 
-    //calculation for output token (x = yout * poolX / poolY - yOut)
+    //calculation for input token x_in = (yout * poolX  /poolY* fee) / (1 - yOut/poolY * fee)
+    //where fee is the standard 1021/1024 ratio.
```

Comment:
comments are not accurate.
Here your fee is 1024/1021.
Please change it and other ok